mbox2html.py

```diff
@@ -46,7 +46,10 @@ def get_header_text(msg, item, default='utf-8'):
             except:
                 header_section = text
         else:
-            header_section = text.decode(charset or default)
+            try:
+                header_section = text.decode(charset or default)
```

Comment:
me doing a `decode(charset or default)` here is superfluous as I check for charset being None in the above if-try. I was looking at more compact examples for header decoding and took the or-construct without reflection. I do like the verbosity of the function to the compact examples I saw